EpiAware/src/utilities.jl

```diff
@@ -1,23 +1,5 @@
-"""
-    scan(f, init, xs)
-
-Apply a function `f` to each element of `xs` along with an accumulator hidden state with intial
```

Comment:
I'm not clear why we would remove this due to another issue and not just update it in that PR? It seems like this information will be useful to have in any work that improves docstrings?

Reply:
Thats a good point.